### ISSUE TITLE
Delegate reports are adopted as verification: a reviewer endorsed the bug, a planner supplied the wrong prior, an explorer handed the parent the oracle's weights

### ISSUE THREAD

**Issue by obra** [prime-radiant-inc/evener#498]
Delegation roughly quadrupled after #492, and the quality of what delegates return is now load-bearing. Three failures in one run came from the parent adopting a delegate's output as verification. In one of them the delegate handed the parent the grader's own private weights, which made fake verification the cheapest option available.

## Scale changed

Across the same 45-task set, 21 hours apart:

| | explorer sessions | subagent sessions |
|---|---|---|
| before (`main-da9146ce1`) | 0 | 28 |
| after (`main492-120bb3b7f`) | **73** | 34 |

Spanning 30 distinct tasks. Before #492 every explorer spawn was refused, so this surface barely existed; now it carries real weight. In one cluster of ten failures, trials with at least one delegate went 12/16 while trials with none went 2/8 — in the baseline the same split was 4/8 versus 8/16, i.e. no signal. Delegation is now doing work, which makes its failure modes worth naming.

## The three failures

**1. A reviewer delegate endorsed the exact bug it was asked to find.** `torch-tensor-parallelism__XCF4Cms`: the implementation has an extra `dist.all_reduce` in both autograd backwards, over-scaling gradients by `world_size`. The `subagent` reviewer looked straight at it and blessed it:

> "Row output all-reduce/backward all-reduce is also the correct adjoint… Replacing its backward with identity would lose contributions from replicated consumers."

That sentence argues against the precise edit that made the passing trial pass. The parent applied the delegate's two cosmetic suggestions and closed.

**2. A planner delegate volunteered the fatal prior, and arrived after commitment.** `torch-pipeline-parallelism__7sL8CbX`: the delegate offered "All forward micros first, then all backward micros (**usually reverse order**)" unprompted, and its report landed the same second the file was already written. The reviewer delegate that might have caught the empty-`losses` branch was `job_stop`ped at 23:14:28 having produced nothing — and the task list recorded that step as "Reviewed the full source."

**3. An explorer delegate dumped the oracle's private parameters into the parent's context.** `model-extraction-relu-logits__VfCWCU2`: before the parent had written a single probe, its explorer reported

```
A1 shape= (20, 10) … values= array([[ 0.17640523, …
```

— the reference implementation's own weights. That made "check my answer against `forward.A1`" the cheapest available verification, and it became the parent's entire strategy. The parent then overfit to the visible 20-neuron network; the graded network has 30.

The delegate stated nothing false. It made the one check that *cannot* detect instance-overfitting free, and free crowded out real verification.

## Why write-blocking does not address this

#492 blocks a read-only delegate's file-tool writes. Nothing about that stops a delegate from *reading* the answer key and handing it upstream in prose. A read-only delegate is exactly as capable of leaking an oracle as a writable one, and the read-only framing may make its output feel safer to consume.

## The contrast that suggests a lever

`torch-tensor-parallelism__3Fgs3WK` passed with the same task and the same underlying bug present in its first draft. The difference: it used `agent_type: "default"` and then called `read_transcript(range="last:20")` to read the delegate's **reasoning** rather than its summary — and that reasoning said the current reductions over-scale. Same information, different channel, opposite outcome.

That points at something concrete: a delegate's summary is a lossy, confidence-flattening rendering of work whose doubts live in the body. Two directions worth weighing:

1. Surface a delegate's hedges and open questions in the delivered result rather than only its conclusion, or make reading the delegate's reasoning the default rather than an extra call the parent has to think of.
2. Treat a delegate report as evidence to be checked rather than a verification that has occurred — the parent currently has no prompt-level or mechanical reason to distinguish "my delegate said it is correct" from "I verified it."

## Two smaller delegate-lifecycle defects from the same run

- **`delegate_send` returns `target_busy` as a hard error** (`XCF4Cms` seq 24, `gpt2-codegolf__zksE9pR` seq 90). Busy is a transient state, not a failure.
- **Reports land after the parent has committed**: `7sL8CbX` at +0s from the file write, `zksE9pR`'s explorer at +452s and its subagent at +751s with 149s of budget left. When the parent had already moved on, the work was pure cost.

Related: #363 (a lost verification channel is silently absorbed), #366 (read-only delegate boundaries), #492 (the change that made this surface load-bearing).

**Comment by obra** [comment]
Classified: **feature (product/prompt policy)** with two real engineering sub-items noted: (1) the `target_busy` error is returned as a hard error from the send path despite being transient and self-describing as retryable — that is a defect worth its own issue and I'll file it; (2) reports landing after the parent committed is inherent to the fire-and-report design. The main body (delegate reports adopted as verification) is prompt/product work. Leaving open for that.

**Comment by obra** [comment]
**Independent triage verification** (at `27503c07dacc`, 2026-09-13).

**Verdict:** `legit-current` — the delegation-quality failure is real and load-bearing
(delegation ~quadrupled after #492), but the failure is the parent adopting a delegate
report as verification: a model/prompt-policy problem, not a code defect. One engineering
sub-item is a genuine defect: `delegate_send` surfaces a transient busy as a hard error.
When steering loses to a settlement, `ReserveStart` returns `errDelegateTargetBusy`
(`agent/delegate_tree_start.go:602-605`) and `stableDelegateSendTool` returns it as a tool
error because `result.DelegateID == ""` (`agent/session_tools.go:165-167`), even though
busy is retryable and self-describing. The other sub-item ("reports land after the parent
committed") is inherent to fire-and-report, not a bug.

**Autonomy:** `needs-human` — the primary work is prompt/product policy: whether the
delegate's hedges/uncertainty are surfaced in the delivered result, and whether a report
is framed as evidence to be checked. That is a doctrine ruling, not a mechanical fix.

**Recommendation:** `needs-decision` — split the `target_busy` hard-error into its own
bounded fix; keep the verification-adoption work as prompt policy.
